components/templates/admin/news/list.tsx

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import Head from "next/head";
+import styled from "styled-components";
+import Footer from "../../../../components/Footer";
+import Image from "next/image";
+import Link from "next/link";
+import {News} from "../../../../models/news";
+import { InferGetStaticPropsType } from 'next';
+
+export default function NewsListTemplate(props: News[]) {
```

Comment:
@duoctvd 
Chổ này nó phải là như vầy
```
export default function NewsListTemplate(props: { newList: News[] }) {
```

`props` nó đại diên cho tất cả data sẽ được truyền qua component, còn cụ thể thì nó có `newList`.
sau này em có thể truyền thêm `user` chẳng hạn thì nó sẽ được thêm vào khai báo của `props`

```
export default function NewsListTemplate(props: { newList: News[]; user?: User }) {
```

Để cho tường minh và tránh dùng `props.xxx` thì sẽ viết như sau

```
export default function NewsListTemplate({ newList }: { newList: News[] }) {
```

Hoặc trường hợp props có nhiều data

```
interface Props {
  newList: News[];
  user?: User 
};

export default function NewsListTemplate({ newList, user }: Props) {
```

Reply:
@bit-thuybit: thank you very much. Em sẽ lưu lại cái này để hiểu rõ hơn về prop! em đọc lúc trưa nhưng giờ đọc cái này của chị em hiểu rõ hơn vài thứ!

Reply:
> @bit-thuybit: thank you very much. Em sẽ lưu lại cái này để hiểu rõ hơn về prop! em đọc lúc trưa nhưng giờ đọc cái này của chị em hiểu rõ hơn vài thứ!

@bit-thuynt: bổ sung, giờ em làm như trên và yarn build đã hết lỗi rồi! em sẽ tiếp tục với edit form

Reply:
@duoctvd em cứ làm tiếp cái edit form đi, ko cần deploy ngay đâu.